automated install/basic-install.sh

```diff
@@ -245,7 +245,7 @@ if is_command apt-get ; then
     fi
     # Since our install script is so large, we need several other programs to successfully get a machine provisioned
     # These programs are stored in an array so they can be looped through later
-    INSTALLER_DEPS=(apt-utils dialog debconf dhcpcd5 git "${iproute_pkg}" whiptail)
+    INSTALLER_DEPS=(apt-utils debconf dhcpcd5 git "${iproute_pkg}" whiptail)
```

Comment:
@bcambl 
`debconf` is then not required as well. It is the actual package that provides `debconf-apt-progress` command.

Reply:
I was thinking the same but I did not want to remove the packages in the event they are required for a particular minimal Debian based distro.  
The two packages are available by default in my Debian 10 test environment.

The following commands are provided by the packages:
```
bcambl@debian10:~$ dpkg-query -L debconf | grep bin
/usr/bin
/usr/bin/debconf
/usr/bin/debconf-apt-progress
/usr/bin/debconf-communicate
/usr/bin/debconf-copydb
/usr/bin/debconf-escape
/usr/bin/debconf-set-selections
/usr/bin/debconf-show
/usr/sbin
/usr/sbin/dpkg-preconfigure
/usr/sbin/dpkg-reconfigure


bcambl@debian10:~$ dpkg-query -L apt-utils | grep bin
/usr/bin
/usr/bin/apt-extracttemplates
/usr/bin/apt-ftparchive
/usr/bin/apt-sortpkgs
bcambl@debian10:~$ 

```
I would like to hear from the other team members before removing them as well.
Thanks for the review :)